src/routes/tenant.js

```diff
@@ -29,7 +29,7 @@ protected_route.put('/network-settings', compose([tenantRoute, tenantAdminGuard]
 protected_route.get('/registry/sign-ups', compose([tenantRoute, tenantAdminGuard]), tenant.getSignUpRequests);
 protected_route.put('/registry/sign-ups/approve', compose([tenantRoute, tenantAdminGuard]), tenant.approveSignUpRequest);
 protected_route.put('/registry/sign-ups/reject', compose([tenantRoute, tenantAdminGuard]), tenant.rejectSignUpRequest);
-protected_route.post('/registry/sign-up', compose([tenantRoute, tenantAdminGuard]), auth.signUp);
+// protected_route.post('/registry/sign-up', compose([tenantRoute, tenantAdminGuard]), auth.signUp);
```

Comment:
Why we removed this method? As far as I know, we use it when we register a user from the UI admin panel. This feature is used in TTO app and other product